cmd/CMakeLists.txt

```diff
@@ -42,6 +43,17 @@ add_executable(kv_seek kv_seek.cpp ${SILKRPC_SRC})
 target_include_directories(kv_seek PRIVATE ${CMAKE_SOURCE_DIR})
 target_link_libraries(kv_seek absl::flags_parse gRPC::grpc++_unsecure protobuf::libprotobuf silkworm)
 
+# test only client
+file(GLOB_RECURSE GENERATED_SOURCES CONFIGURE_DEPENDS "${CMAKE_BINARY_DIR}/*.cc")
+message(STATUS "CMAKE_BINARY_DIR: " ${CMAKE_SOURCE_DIR})
+message(STATUS "CMAKE_BINARY_DIR: " ${CMAKE_BINARY_DIR})
+message(STATUS "GENERATED_SOURCES: " ${SILKRPC_SOURCES})
+message(STATUS "GENERATED_SOURCES: " ${GENERATED_SOURCES})
```

Comment:
We could remove the `message` logs (maybe added for debug) in order to maintain the build output as clean as possibile